utils/guidelines_xslt/odd2html/functions.xsl

```diff
@@ -469,8 +469,10 @@
         
         <xsl:variable name="contributors">
             <xsl:variable name="raw.contributors" as="node()*">
-                <xsl:sequence select="tools:retrieveData($docs.repo.contributors)/child::json:array/json:map"/>
-                <xsl:sequence select="tools:retrieveData($spec.repo.contributors)/child::json:array/json:map"/>
+                <xsl:if test="$retrieve-contributors ne 'false'">
```

Comment:
Then this could be simplified into
```suggestion
                <xsl:if test="$retrieve-contributors">
```

Reply:
Problem is that in my experience, it doesn't work reliably to pass in values other than strings. Particularly boolean values seem to have problems. I know it's clumsy to do it like this, but it was intentional to avoid nasty problems which are really hard to debug.

Reply:
obviously, the same applies to the other suggestion above.

Reply:
The funny thing is, that almost every value passed evaluates to `true()`, e.g., non-empty strings, integers other than 0. If you insist in using string comparison, it should not mimic a boolean. Then better to use "no" or something similar.

Reply:
I don't insist on anything, I just don't want to write broekn code that's hard to debug. I don't mind to call it "no", if that is more clear. Normally, I try to avoid boolean-like parameters completely instead, but this name was proposed already, so I did followed my normal procedure – which is to explicitly use "false", because I then have clearly expected values and legible code, which just behaves a bit oddly and looks a bit uninformed, where in fact it's just the opposite.

Reply:
From my perspective this _is_ broken code, but who am I to judge.